### PR TITLE
Remove install plugins / themes in Cloud environments

### PR DESCRIPTION
**Pull request by joehoyle**
I think for subtle reasons, and practically, it's better not to have these available.

### CODE REVIEW

File: inc/namespace.php [hunk 1]
```diff
@@ -216,3 +220,20 @@ function load_plugins() {
 		require dirname( __DIR__ ) . '/plugins/' . $file;
 	}
 }
+
+/**
+ * Disable the install plugins / themes capability in Cloud, as it's not possible to do in that context.
+ *
+ * @param array $caps
+ * @param string $cap
```
Comment (rmccue) [inline inc/namespace.php]: As @roborourke mentioned, these should be documented.

File: inc/namespace.php [hunk 2]
```diff
@@ -216,3 +220,20 @@ function load_plugins() {
 		require dirname( __DIR__ ) . '/plugins/' . $file;
 	}
 }
+
+/**
+ * Disable the install plugins / themes capability in Cloud, as it's not possible to do in that context.
```
Comment (rmccue) [inline inc/namespace.php]: This is too long, split to short description + long desc.

Comment (roborourke) [reply inc/namespace.php]:
```suggestion
 * Disable the install plugins / themes capability.
 * 
 * In the cloud context it's not possible to do this so this hides 
 * non functional UI.
```

File: inc/namespace.php [hunk 3]
```diff
@@ -216,3 +220,20 @@ function load_plugins() {
...
+ *
+ * @param array $caps
```
Comment (roborourke) [inline inc/namespace.php]:
```suggestion
 * @param array $caps The required capabilities list.
```

File: inc/namespace.php (hunk 1 above)
Comment (roborourke) [inline inc/namespace.php]:
```suggestion
 * @param string $cap The current capability being checked.
```

**Pull request merged**